hathor/simulator/heap_clock.py

```diff
@@ -94,10 +95,3 @@ class MemoryReactorHeapClock(MemoryReactor, HeapClock):
     def __init__(self):
         MemoryReactor.__init__(self)
         HeapClock.__init__(self)
-
-    def run(self):
```

Comment:
Why did you remove this method?

Reply:
It was not removed, it was moved to `MemoryReactor` in `hathor/reactor/memory_reactor.py` and is available in this class via inheritance. Before, we had `TestMemoryReactorClock` with a duplicate method. I did this to unify them.

Reply:
Got it. Thanks!